File: service/src/main/java/net/huray/backend/minuting/entity/MinutesAttendee.java
package net.huray.backend.minuting.entity;

import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.Id;
import javax.persistence.IdClass;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Entity(name = "minutes_attendee")
@IdClass(MinutesAttendeeId.class)
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class MinutesAttendee {

    @Id
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "minutes_id")
    private MinutesEntity minutes;

    @Id
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "member_id")
    private Member member;

}

File: domain/src/main/kotlin/net/huray/backend/minuting/entity/MinutesAttendee.kt
package net.huray.backend.minuting.entity

import javax.persistence.*

@Entity(name = "minutes_attendee")
@IdClass(MinutesAttendeeId::class)
class MinutesAttendee {

    @Id
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "minutes_id")
    lateinit var minutes: MinutesEntity

    @Id
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "member_id")
    lateinit var member: MemberEntity

}